### PR TITLE
-Add information for RMQSignal feature

### PR DESCRIPTION
**Pull request by milanac030988**

### CODE REVIEW

File: config/robotframework_aio/release_items_QConnectBase.json [hunk 1]
```diff
@@ -45,9 +45,18 @@
...
+**RMQSignal**
+ 
+* RMQSignal provides a synchronization mechanism designed to facilitate inter-process communication (IPC) between Robot Framework test processes using RabbitMQ as the underlying message broker. 
+This module leverages RabbitMQ's message queues to enable communication and coordination between different test processes, allowing them to exchange signals, notifications, or other relevant information.
+ 
+**Environment/Installation**
+ 
+* For using RMQSignal, required to run the
+file QConnectBase/tools/setup rabbitmq.bat to install the entire infrastructure required for these Auxiliary Utilities.
+"
```
Comment (HolQue) [inline config/robotframework_aio/release_items_QConnectBase.json:59]: Hi Cuong,

tiny issue only:

Lines after an item (*) need to have the same indentation like the lines containing the item (to indicate that they belong together).

Not:

```
* item line 1
item line 2
```

But:

```
* item line 1
  item line 2
```

**Review by HolQue — CHANGES_REQUESTED** [review]
https://github.com/test-fullautomation/robotframework-qconnect-base/pull/41/files#r1458690550

File: config/robotframework_aio/release_items_QConnectBase.json (hunk 1 above)
Comment (test-fullautomation) [reply config/robotframework_aio/release_items_QConnectBase.json:59]: done...

**Review by test-fullautomation — APPROVED** [review]
Hi Cuong,
looks good to me!
Thank you,
 Thomas

**Pull request merged**